third_party/mkl/build_defs.bzl

```diff
@@ -15,18 +15,25 @@ mkl_repository depends on the following environment variables:
 _TF_MKL_ROOT = "TF_MKL_ROOT"
 
 def if_mkl(if_true, if_false = []):
-    """Shorthand for select()'ing on whether we're building with MKL.
+    """Shorthand for select()'ing on whether we're building with oneDNN.
+      OneDNN gets built if we are building on platforms that support oneDNN
+      (x86 linux/windows) or if specifcially configured to use oneDNN.
 
     Args:
-      if_true: expression to evaluate if building with MKL.
-      if_false: expression to evaluate if building without MKL.
+      if_true: expression to evaluate if building with oneDNN.
+      if_false: expression to evaluate if building without oneDNN.
 
     Returns:
       a select evaluating to either if_true or if_false as appropriate.
     """
     return select({
         "@org_tensorflow//third_party/mkl:build_with_mkl": if_true,
```

Comment:
Should we delete this one?

Reply:
We think that ARM may need it when compiling with oneDNN. We will probably remove it later with refactoring.

Reply:
Got it. Could you please add a comment explaining this in the code as well? Thank you!

Reply:
Comment will be added.